pkg/parser/ast/dml.go

```diff
@@ -281,7 +281,8 @@ type TableName struct {
 	PartitionNames []model.CIStr
 	TableSample    *TableSample
 	// AS OF is used to see the data as it was at a specific point in time.
-	AsOf *AsOfClause
+	AsOf    *AsOfClause
+	IsAlias bool
```

Comment:
Better add some comments and examples for this field.

Reply:
have updated.